pypdf/_writer.py

```diff
@@ -1564,7 +1565,7 @@ def get_threads_root(self) -> ArrayObject:
         """
         The list of threads.
 
-        See §8.3.2 from PDF 1.7 spec.
+        See §12.4.3 of the PDF 1.7 or PDF 2.0 specification.
```

Comment:
See above.

Reply:
The previous 8.3.2 is Coordinate Spaces; whether this has anything to do with threads I do not know. I changed it to 12.4.3 which is Articles.